api/sheets/get_name.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Get the name of a Data Commons node given its dcid.
+nav_order: 4
+parent: SHEETS
+grand_parent: API
+---
+
+# Get the name of a node.
+
+## =c(dcids)
+
+Given a cell or a column range of cells with DCIDs, return the names of those dcids.
+
+**Arguments**
+*    `dcids (str)` - DCIDs to get the names of
```

Comment:
this is a python style annotation. In sheets I guess there is no strict way of typing?

Reply:
Yeah, I got rid of all the type annotations.